packages/sync/src/Functions.php

```diff
@@ -209,6 +272,11 @@ public static function get_raw_or_filtered_url( $url_type ) {
 		return self::get_protocol_normalized_url( $url_function, $url );
 	}
 
+	/**
+	 * Return the home_url.
```

Comment:
Should we clarify that this home URL is in escaped form?

Reply:
ecbf5af85